assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -492,6 +492,13 @@ $cart-item-label-offset:                $cart-thumbnail-maxWidth + $cart-item-sp
             float: right;
         }
     }
+
+    .checkout-multiple {
```

Comment:
`cart-actions` means that `actions` is an element of the `cart`. There's no need for camel case in that instance :)
See `cart-additionalCheckoutButtons` in the same file.